include/vrt.h

```diff
@@ -494,14 +496,16 @@ struct vmod_data {
 };
 
 /***********************************************************************
- * VCL events sent to VMODs
+ * VCL events sent to VMODs and directors
  */
 
 enum vcl_event_e {
 	VCL_EVENT_LOAD,
 	VCL_EVENT_WARM,
 	VCL_EVENT_COLD,
 	VCL_EVENT_DISCARD,
+	// Only for directors
+	VDI_EVENT_SICK,
 };
```

Comment:
I would drop the "Only for directors" comment because in order to preserve the ABI we should not group events by theme and rather append them to the enum. We can instead list types of events in the comment block above:

```c
/***********************************************************************
 * VCL events sent to VMODs and directors:
 * - VCL: VCL temperature events
 * - VDI: director health events
 */
```

Or maybe something a bit more elaborated.

Reply:
The comment could still be changed later, it does not break the ABI ;) But yea, good point!